q2_types/sample_data/tests/test_transformer.py

```diff
@@ -30,11 +30,24 @@ def test_pd_series_to_alpha_diversity_format(self):
 
     def test_alpha_diversity_format_to_pd_series(self):
         filename = 'alpha-diversity.tsv'
-        input, obs = self.transform_format(AlphaDiversityFormat, pd.Series,
-                                           filename)
-        input = pd.Series.from_csv(str(input), sep='\t', header=0)
+        _, obs = self.transform_format(AlphaDiversityFormat, pd.Series,
+                                       filename)
 
-        assert_series_equal(input, obs)
+        exp_index = pd.Index(['Sample1', 'Sample4'], dtype=object)
+        exp = pd.Series(['0.9709505944546688', '0.7219280948873623'],
```

Comment:
No, the old behavior was to cast the values to floats, now they remain as an `object` dtype. Do you think busywork will catch any other plugins where we might have been assuming that casting behavior?

For reference, the test results if the series is defined as `[0.9709505944546688, 0.7219280948873623]`, instead (floats vs of strings):

```bash
E       AssertionError: Series are different
E
E       Series values are different (100.0 %)
E       [left]:  [0.9709505944546688, 0.7219280948873623]
E       [right]: [0.9709505944546688, 0.7219280948873623]
```